File: javascript/p006.js
function main() {
	let total = 0;
	let total2 = 0;
	for (let i = 1; i <= 100; i++) {
		total += i**2;
		total2 += i;
	};
	return ((total2**2) - total);
};

console.log(main());

File: lua/p006.lua
function main() 
	total = 0
	total2 = 0
	for i=1, 100, 1 do
		total = total + math.pow(i, 2)
		total2 = total2 + i
	end
	return (math.pow(total2, 2) - total)
end

print(main())